README.md

```diff
@@ -117,7 +117,9 @@ See [the examples](./examples/README.md) for more detailed usage or read about a
 * `updatePassedSnapshot`: (default `false`) Updates a snapshot even if it passed the threshold against the existing one.
 * `blur`: (default `0`) Applies Gaussian Blur on compared images, accepts radius in pixels as value. Useful when you have noise after scaling images per different resolutions on your target website, usually setting its value to 1-2 should be enough to solve that problem.
 * `runInProcess`: (default `false`) Runs the diff in process without spawning a child process.
-* `dumpDiffToConsole`: (default `false`) Will output base64 string of a diff image to console in case of failed tests (in addition to creating a diff image). This string can be copy-pasted to a browser address string to preview the diff for a failed test.
+* `dumpDiffToConsole`: (default `false`)
```

Comment:
It seems odd to have the options be `true`, `false`,`inline` doesn't it? Since there is both a boolean and a string mixed in.

Can we instead detect if running in iTerm and if so display the image inline if `dumpDiffToConsole` is true?

Or does this bloat the output too much? Can you share screenshots of what it would look like?

Reply:
Hey Andres,

Here's what it looks like in iTerm,

<img width="600" alt="Screenshot 2020-11-10 at 09 26 08" src="https://user-images.githubusercontent.com/461449/98655133-0a6cad80-2337-11eb-8430-833c3c82c5c3.png">

With regards to mixing `boolean` and `inline`, I mostly just wanted to have an explicit option that re-used the exiting glue code. As for the relative oddness of the type itself, I guess that's up to you to decide, but the type is representable in the TypeScript definitions without any problems.

### Detection

It might be possible to detect support for the Inline Images Protocol, but I'd prefer to do it based on the feature itself rather than the just checking if the Terminal is iTerm. This is so that the feature still works for other Terminals that support it (the options are slim at the moment).

This might add a lot of code that's full of OSC escape sequences and would likely be a bit of a maintenance burden.

Thoughts?

Reply:
What about a new option, `enableInlineConsoleDiff` or something to that effect?

Reply:
Hmm, yeah I think either a new option like @igetgames is suggesting (there are so many now!) might work!

@americanexpress/one what do you all think? New flag or mix booleans with string in the existing option?

Reply:
Hey folks,

I've updated the MR to add a new option: `dumpInlineDiffToConsole`, the existing `dumpDiffToConsole` is left as-is.
In addition if the terminal isn't supported (or `ENABLE_INLINE_DIFF` is in the env), we fallback to `dumpDiffToConsole`.